crates/sui-protocol-config/src/lib.rs

```diff
@@ -1512,6 +1534,15 @@ impl ProtocolConfig {
                     // zklogin::check_zklogin_issuer
                     cfg.check_zklogin_issuer_cost_base = Some(200);
                 }
+                28 => {
+                    if chain != Chain::Mainnet && chain != Chain::Testnet {
+                        cfg.feature_flags.throughput_aware_consensus_submission = true;
+                        cfg.feature_flags.consensus_throughput_profile_ranges = vec![0, 2_000];
+                    } else {
+                        // Mainnet and testnet
+                        cfg.feature_flags.consensus_throughput_profile_ranges = vec![0, 1_000];
```

Comment:
Up to 1K tx/sec we'll use the enhanced tx duplication  - over the 1K/sec we'll fallback to default settings.